### ISSUE TITLE
High density issues for Target Size (Minimum) WCAG 2.2

### ISSUE THREAD

**Issue by jake-costa** [jpmorganchase/salt-ds#5471]
A number of components when using HD will fail Target Size (Minimum) WCAG 2.2 requirement.

This also doesn't appear to be a straightforward way to address.

Discussion is needed.

**Comment by joshwooding** [comment]
Guidance for High density WCAG conformance is now live on the site
